### ISSUE TITLE
Where to attach the dependents of a MWE

### ISSUE THREAD

**Issue by sylvainkahane** [surfacesyntacticud/guidelines#41]
We have two different practices concerning the dependents of MWE in SUD.

1) In GSD, dependents are always attached to the head of the MWE. It is also what is done in non-native SUD treebank due to the lack of an internal analysis of the MWE.

2) In French spoken treebanks we decided to favor a surface-syntactic analysis when the syntax is regular. For instance, in _à partir de_ X, _à partir_ is analyzed as a MWE and _de_ X depends on _partir_: http://universal.grew.fr/?custom=64137cfda75bb (some analyses are incoherent and must be revised, by the way).

I think that the second choice is more coherent with the SUD principles. In issue #37, @Valentin-D-Richard propose to analyze _est-ce que_ as a MWE. If this is done, I think the dependent must be attached to _que_, as it is now. The decision to analyze _est-ce que_ as a MWE or not must be independent of its syntactic analysis.

**Comment by perrier54** [comment]
I agree with @sylvainkahane . If I understand his proposal correctly, we annotate the example _est-ce que tu viens ?_ (do you come ?) in this way : 
![sud_ex](https://user-images.githubusercontent.com/29424639/226096063-3af96cd3-89f0-4b66-acf4-4a160d4689fc.svg)

**Comment by sylvainkahane** [comment]
@perrier54 yes, that's the proposition. We need to add an `ExtPos`. All idioms should have an `ExtPos` I think, even when it is equal to the head POS.
If we agree on this proposal, we must add it in the guidelines and revise all MWEs with a dependent. Some Grew rules can be written for regular and common MWEs such as ADP NOUN.

**Comment by perrier54** [comment]
Just a small remark: in the MWE _est-ce que_, _être_ does not behave as an auxiliary (no clitic climbing). I therefore propose `ExtPos=VERB`. 
The annotation of _est-ce que tu viens ?_ becomes:
![sud_ex](https://user-images.githubusercontent.com/29424639/226160356-3aada513-ccbc-4950-a827-baa8b5b1bb16.svg)

**Comment by sylvainkahane** [comment]
_est-ce que_ /esk/ as a whole does not behave as a verb. 
It seems that /esk/ can be added quite freely in interrogative sentences:
- _tu viens demain ?_ vs _esk tu viens demain ?_
- _où ça se passe ?_ vs _où esk ça se passe ?_

It is why I feel like annotating esk as `ExtPos=PART` depending on the main verb by a relation `advmod`. 
On the other way, it must be noted that esk does not combine with the verb in case of inversion:
- _peux-tu venir?_ vs _*esk peux-tu venir?_

Therefore esk is not a pure dependent. The inversion and esk are markers in competition. But `advmod` is also used for negation _pas_ which also has effect on the rest of the construction:
- _je n'ai pas bu d'alcool_ vs *_j'ai bu d'alcool_

**Comment by perrier54** [comment]
The `ExtPos` feature does not change when converting from SUD to UD.
If we annotate /esk/ with `ExtPos=PART`, this imply that  in SUD, a PART is the head of a sentence with a `comp:obj` dependent, which is very strange.

**Comment by perrier54** [comment]
For all MWE with an internal syntactic structure, external dependencies must be attached to the element of the MWE that is consistent with the syntax of the whole sentence.
The label of the external dependency must take into account the POS of the MWE.

Example:  _la proximité de l'école l'arrange_ (the proximity of the school suits him)  _proximité -[udep]-> de_
_Il habite à proximité de l'école_ (He lives near the school ) _proximité -[comp:obl]-> de_
